README.md

```diff
@@ -404,18 +404,25 @@ You can read about [getting started](https://julialang.org/manual/getting-starte
 
 The following distributions include julia, but the versions may be out of date due to rapid development:
 
+<a href="https://repology.org/metapackage/julia">
+    <img src="https://repology.org/badge/vertical-allrepos/julia.svg" alt="Packaging status" align="right">
```

Comment:
Unfortunately, there's no way to know whether a version is development or an official one. So it's just showing the latest one. There's one improvement planned which will hide gentoo's "9999" version, but for nixpkgs there's no way to fix it.